src/librustc/mir/interpret/value.rs

```diff
@@ -415,48 +415,60 @@ impl<'tcx, Tag> Scalar<Tag> {
         }
     }
 
+    fn to_unsigned_with_bit_width(self, bits: u64) -> InterpResult<'static, u128> {
+        let sz = Size::from_bits(bits);
+        self.to_bits(sz)
+    }
+
+    // Consume the scalar to produce an `u8`. It fails if the scalar is a pointer.
```

Comment:
"Consume" isn't really relevant here, so it shouldn't be what the docs lead with. Also the grammar is inconsistent between the two sentences.

What about: "Converts the scalar to produce a `u8`. Fails if the scalar is a pointer."

Reply:
Also also make sure these are all actually doc comments: They should start with `///`

Reply:
Done :P